package.json

```diff
@@ -28,9 +28,10 @@
     "build:only": "tsc && cp package.json build && browserify src/index.ts -p tsify -d -s vl | exorcist build/vega-lite.js.map > build/vega-lite.js",
     "postbuild": "node node_modules/uglify-js/bin/uglifyjs build/vega-lite.js -cm --source-map content=build/vega-lite.js.map,filename=build/vega-lite.min.js.map -o build/vega-lite.min.js && npm run schema",
     "build:examples": "npm run data && npm run build:only && npm run build:examples-only",
-    "build:examples-only": "npm run build:example && npm run build:examples-normalized",
+    "build:examples-only": "TZ=America/Los_Angeles scripts/build-examples.sh && npm run build:examples-normalized",
+    "build:examples-full": "TZ=America/Los_Angeles scripts/build-examples.sh 1",
```

Comment:
Instead of adding one more command, we should output generated vega version and check the version instead. (We only need to force re-build when we change vega version.)